recipe/meta.yaml

```
@@ -5,14 +5,14 @@ package:
   version: {{ version }}
 
 source:
-  git_url: https://github.com/cyclus/cyclus.git
-  git_rev: 1.6.0
+  git_url: https://github.com/bennibbelink/cyclus.git
+  git_rev: cleanup-dependencies
 
 build:
   number: 3
```

If you intend to publish this, don't forget to bump the build number
```suggestion
  number: 4
```